redis/src/main/scala/zio/redis/Input.scala

```diff
@@ -591,4 +613,13 @@ object Input {
     def encode(data: Boolean)(implicit codec: Codec): Chunk[RespValue.BulkString] =
       Chunk.single(encodeString(if (data) "YES" else "NO"))
   }
+
+  trait Implicits {
+    implicit val bytesEncoder: Input[Chunk[Byte]] = ByteInput
+    implicit val booleanInput: Input[Boolean]     = BoolInput
+    implicit val stringInput: Input[String]       = StringInput
+    implicit val longInput: Input[Long]           = LongInput
+  }
+
+  object Implicits extends Implicits
```

Comment:
This seems to be used only in tests. If that's the case, please define them there instead.